Cython/Distutils/extension.py

```diff
@@ -95,10 +99,6 @@ def __init__(self, name, sources,
         self.no_c_in_traceback = no_c_in_traceback
         self.cython_compile_time_env = cython_compile_time_env
 
-# class Extension
-
-read_setup_file = _Extension.read_setup_file
-
```

Comment:
This is API. I don't think it should be removed.

Reply:
Problem is that in setuptools I was not able to find equivalent function.

Reply:
Let's do it that way.